server/game/cards/08-MotC/Gossip.js

```diff
@@ -0,0 +1,37 @@
+const DrawCard = require('../../drawcard.js');
+const { Players, PlayTypes } = require('../../Constants');
+
+class Gossip extends DrawCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'Name a card that your opponent cannot for for the phase',
```

Comment:
"cannot for for"

Reply:
oops I changed it, thanks for catching it